src/services/account/useAccountDetails.ts

```diff
@@ -55,16 +66,7 @@ export const PaymentMethodSchema = z
     }),
     billingDetails: z
       .object({
-        address: z
-          .object({
-            city: z.string().nullable(),
-            country: z.string().nullable(),
-            line1: z.string().nullable(),
-            line2: z.string().nullable(),
-            postalCode: z.string().nullable(),
-            state: z.string().nullable(),
-          })
-          .nullable(),
+        address: AddressSchema,
```

Comment:
`l`: Can you add `.nullable()` here as well, makes it a tad easier to know whether it is or not without having to look back at the schema itself

Reply:
For sure! I don't really mind, there's no issues with .nullable().nullable() right?

I originally wanted to keep the schemas standalone which is why I had the nullable there only

Reply:
Yea there's no issues with doubling it up!

Reply:
Updated in https://github.com/codecov/gazebo/pull/2955/commits/39fa1ffc002ae2871b18ccbb4b64538146d5366e